Description: Upstream changes introduced in version 1.5.3-2.8
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 cups (1.5.3-2.8) unstable; urgency=low
 .
   * Bugfix the configuration stanzas split: it left some possibilities
     to have configuration stanzas taken from the wrong file.
     - Refresh affected patches too.
 .
 The person named in the Author field signed this changelog entry.
Author: Didier Raboud <odyx@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- cups-1.5.3.orig/scheduler/conf.c
+++ cups-1.5.3/scheduler/conf.c
@@ -154,7 +154,20 @@ static const cupsd_var_t	cupsd_vars[] =
   { "RIPCache",			&RIPCache,		CUPSD_VARTYPE_STRING },
   { "RootCertDuration",		&RootCertDuration,	CUPSD_VARTYPE_INTEGER },
   { "ServerAdmin",		&ServerAdmin,		CUPSD_VARTYPE_STRING },
+  { "ServerBin",		&ServerBin,		CUPSD_VARTYPE_PATHNAME },
+#ifdef HAVE_SSL
+  { "ServerCertificate",	&ServerCertificate,	CUPSD_VARTYPE_PATHNAME },
+#  if defined(HAVE_LIBSSL) || defined(HAVE_GNUTLS)
+  { "ServerKey",		&ServerKey,		CUPSD_VARTYPE_PATHNAME },
+#  endif /* HAVE_LIBSSL || HAVE_GNUTLS */
+#endif /* HAVE_SSL */
   { "ServerName",		&ServerName,		CUPSD_VARTYPE_STRING },
+  { "ServerRoot",		&ServerRoot,		CUPSD_VARTYPE_PATHNAME },
+  { "SMBConfigFile",		&SMBConfigFile,		CUPSD_VARTYPE_STRING },
+  { "StateDir",			&StateDir,		CUPSD_VARTYPE_STRING },
+#ifdef HAVE_AUTHORIZATION_H
+  { "SystemGroupAuthKey",	&SystemGroupAuthKey,	CUPSD_VARTYPE_STRING },
+#endif /* HAVE_AUTHORIZATION_H */
   { "Timeout",			&Timeout,		CUPSD_VARTYPE_INTEGER },
   { "UseNetworkDefault",	&UseNetworkDefault,	CUPSD_VARTYPE_BOOLEAN },
   { "WebInterface",		&WebInterface,		CUPSD_VARTYPE_BOOLEAN },
